src/modules/oneProtectedData/BuyBlock.tsx

```diff
@@ -12,14 +14,21 @@ export function BuyBlock({
   salePriceInNRLC: number;
 }) {
   const queryClient = useQueryClient();
+  const { address } = useUserStore();
```

Comment:
use existing function => myCollectionsQuery
```suggestion
  const { address } = useUserStore();
    const {
    isLoading,
    isSuccess,
    data: collections,
    isError,
    error,
  } = useQuery(myCollectionsQuery({ address: address! }));
```